src/future-hopr-lib-components/Navbar/navBar.tsx

```diff
@@ -138,13 +138,13 @@ const NavBar: React.FC<{
               <MenuIcon />
             </IconButton>
             <Logo className="logo-hopr">
-              <a href="/">
+              {/* <a href="/"> */}
```

Comment:
state was lost due to `a` tag, maybe a Link from react router would solve this?

Reply:
We can do it